javascript-modal/main.js

```diff
@@ -0,0 +1,16 @@
+const popup = document.getElementById('myPopup');
+const btn = document.getElementById('mybtn');
+const span = document.getElementsByClassName('close')[0];
```

Comment:
If you just did a normal `querySelector` there would be no need for the `[0]`. You're returning a list of all `close` elements, but there's only one. If you just did `const $span = document.querySelector('.close')` it would return only one